src/helpers/helpers.js

```diff
@@ -24,4 +25,5 @@ export default {
   VolumeRendering,
   Voxel,
   X,
+  EventBasedProgressBar,
```

Comment:
Can you please rename it ProgressBarEventBased and move it up alphabetically? @chenkaiC4